OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs

```diff
@@ -2876,6 +2876,22 @@ internal static void UpgradeActorRules(int engineVersion, ref List<MiniYamlNode>
 					}
 				}
 
+				if (engineVersion < 20160109)
+				{
+					//Divide levels by 100
+					if (depth == 1 && (node.Key == "GainsExperience" || node.Key.StartsWith("GainsExperience@"))
+							&& node.Value.Nodes.FirstOrDefault(n => n.Key == "Upgrades") != null)
+					{
+						foreach (var kv in node.Value.Nodes.Where(n => n.Key == "Upgrades")) {
+							var key = float.Parse(kv.Key) / 100;							
+							// t is a MiniYamlNode that looks like ("200", "firepower, damage, speed, reload, inaccuracy, rank")
+							// The key of this needs to be divided by 100 and saved.
+							var newKv = new MiniYamlNode(key.ToString(), kv.Value);
```

Comment:
We do it in other places though, for example [here](https://github.com/OpenRA/OpenRA/blob/7729646f45ea254d7f7e36666893d9b00868120c/OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs#L433-L435) or [here](https://github.com/OpenRA/OpenRA/blob/7729646f45ea254d7f7e36666893d9b00868120c/OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs#L1577-L1578), or am I missing something again?